include/sharg/enumeration_names.hpp

```diff
@@ -42,6 +42,9 @@ namespace sharg::custom
  * \ref tutorial_parser for an example of customising a type within your own namespace.
  *
  * \remark For a complete overview, take a look at \ref parser
+ *
+ * \details
+ * \experimentalapi{Experimental since version 1.0.}
```

Comment:
Do we really need a since here? It would only show how long its taking us but doesn't help much?

Reply:
This is the way in seqan3. The API should not be dependent on the patch version.

Reply:
I didn't mean the patch level hear.
I'd do this:
```suggestion
 * \experimentalapi{Experimental}
```

Such that if we are in version 5.12 it doesn't show that this is STILL experimental from version 1.0 
But no strong feelings.

Reply:
I didn't mean the patch level hear.
I'd do this:
```suggestion
 * \experimentalapi{Experimental}
```

Such that if we are in version 5.12 it doesn't show that this is STILL experimental from version 1.0 
But no strong feelings.

Reply:
I think it's helpful if it was no api before. And for new features. In the CMake documentation you have "Available since 3.22" and so on.

Reply:
alright. let's keep it then